adapters/secta/src/sdk/positionDetails.ts

```diff
@@ -0,0 +1,51 @@
+function getTickAtSqrtPrice(sqrtPriceX96: bigint): number {
+    const Q96: bigint = 2n ** 96n;
+    return Math.floor(Math.log(Number(sqrtPriceX96 / Q96) ** 2) / Math.log(1.0001));
+}
+
+export const getTokenAmounts = async(
+    liquidity: bigint,
+    sqrtPriceX96: bigint,
+    tickLow: number,
+    tickHigh: number,
+    Decimal0: number,
+    Decimal1: number
+)=> {
+    const Q96: bigint = 2n ** 96n;
+    const sqrtRatioA: number = Math.sqrt(1.0001 ** tickLow);
+    const sqrtRatioB: number = Math.sqrt(1.0001 ** tickHigh);
+    const currentTick: number = getTickAtSqrtPrice(sqrtPriceX96);
+    const sqrtPrice: number = Number(sqrtPriceX96 / Q96);
+    let amount0: bigint = 0n;
+    let amount1: bigint = 0n;
+
+    // print all the values
+    console.log("liquidity : " + liquidity.toString());
+    console.log("sqrtPriceX96 : " + sqrtPriceX96.toString());
+    console.log("tickLow : " + tickLow);
+    console.log("tickHigh : " + tickHigh);
+    console.log("Decimal0 : " + Decimal0);
+    console.log("Decimal1 : " + Decimal1);
+    console.log("sqrtRatioA : " + sqrtRatioA);
+    console.log("sqrtRatioB : " + sqrtRatioB);
+    console.log("currentTick : " + currentTick);
+    console.log("sqrtPrice : " + sqrtPrice);
+    
+
+    if (currentTick < tickLow) {
+        amount0 = BigInt(Math.floor(Number(liquidity) * ((sqrtRatioB - sqrtRatioA) / (sqrtRatioA * sqrtRatioB))));
+    } else if (currentTick >= tickHigh) {
+        amount1 = BigInt(Math.floor(Number(liquidity) * (sqrtRatioB - sqrtRatioA)));
+    } else if (currentTick >= tickLow && currentTick < tickHigh) {
+        amount0 = BigInt(Math.floor(Number(liquidity) * ((sqrtRatioB - sqrtPrice) / (sqrtPrice * sqrtRatioB))));
+        amount1 = BigInt(Math.floor(Number(liquidity) * (sqrtPrice - sqrtRatioA)));
+    }
```

Comment:
hi, we are looking to remove inactive liquidity.

can you edit this part to not calculate any position that are out of range?

Reply:
I've removed first and second if statements.